src/EFCore.PG.NodaTime/Query/ExpressionTranslators/Internal/NpgsqlNodaTimeMemberTranslatorPlugin.cs

```diff
@@ -153,6 +154,9 @@ public NpgsqlNodaTimeMemberTranslator(ISqlExpressionFactory sqlExpressionFactory
                 // which timestamp cannot be cast into)
                 return null;
 
+            case "TotalHours":
```

Comment:
Since TotalHours (and the other TotalXXX properties) are specific to Duration, we should probably have a different method in this class only translating properties for that, to keep things separate and clear.

Reply:
I'll refactor it this evening when I'm done with work :)